javascript/operators/await.json

```
@@ -103,17 +103,22 @@
                   "notes": "Not supported in CommonJS modules."
                 }
               ],
-              "opera": {
-                "version_added": "73",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "#enable-javascript-harmony"
-                  }
-                ]
-              },
+              "opera": [
+                {
+                  "version_added": "75"
+                },
+                {
+                  "version_added": "73",
```

I don't think this can be switched off now, so…

```suggestion
                  "version_added": "73",
                  "version_removed": "75",
```